packages/core/addon/services/data.js

```diff
@@ -17,6 +17,7 @@ let store = {
 
 export default class DataService extends Service {
   @service cardstackSession;
+  @service cardLocalStorage;
```

Comment:
is this necessary?

Reply:
It is not, and I have since removed it. I also moved the local storage service into cardhost altogether, so this PR introduces no changes to the core package.